Python/pylifecycle.c

```diff
@@ -1727,6 +1727,11 @@ flush_std_files(void)
 
 */
 
+static void
+finalize_free_lists(PyFreeListState *state)
```

Comment:
Following functions will be moved here in the end.
* _PyDict_Fini
* _PyTuple_Fini
* _PySlice_Fini
* _PyFloat_Fini
* _PyContext_Fini